extensions/adobe/experience/decisioning/option-detail.schema.json

```diff
@@ -16,19 +16,34 @@
     "option-detail-datatype": {
       "properties": {
         "xdm:id": {
-          "title": "Option Identifier",
+          "title": "Decision Option Identifier",
           "type": "string",
           "format": "uri-reference",
           "description": "A unique identifier for the decision option."
         },
+        "xdm:version": {
+          "title": "Decision Option Version",
+          "type": "string",
+          "description": "The version that the decision option object was at when the snapshot was taken."
+        },
         "xdm:name": {
           "type": "string",
-          "title": "Option Name",
+          "title": "Decision Option Name",
           "description": "Option name. The name is displayed in various user interfaces."
         },
+        "xdm:tags": {
```

Comment:
Similar question here - is this the formal definition of xdm:tags in all of XDM?

Reply:
The only other usage of "xdm:tags" besides in decisioning/offer management is in "https://ns.adobe.com/xdm/data/datasource" and in campaigns experience events. The latter are exactly the tags introduced by decisioning. 
For datasources the tags are only strings, for decisioning they are defined entities, i.e. they have an identity and two other entities referencing the tags with the same ID really mean to share the tag.